src/transformers/utils/import_utils.py

```diff
@@ -1098,6 +1099,14 @@ def is_mlx_available():
     return _mlx_available
 
 
+def is_tiktoken_available():
+    return _tiktoken_available
```

Comment:
for our usage, let's say 
```suggestion
    return _tiktoken_available and _blobfile_available
```

Reply:
(we only have to use a single wrapper)